docs/guide/install-and-run.asciidoc

```diff
@@ -4,14 +4,14 @@
 To get started using Elastic APM,
 you need to have:
 
-* an Elasticsearch cluster and Kibana (version 5.6 or above)
-* APM Server
-* APM agents installed in your services
+* {apm-agents-ref}/index.html[APM agents] installed in your services
+* {apm-server-ref-v}/index.html[APM Server]
+* An {ref}/index.html[Elasticsearch] cluster (version 5.6+)
+* {kibana-ref}[Kibana] (version 5.6+)
```

Comment:
v6.2 is the minimum supported version of Kibana for use the Elastic APM

Reply:
I actually wouldn't mention versions at all; if anything, just encourage users to have apm server + elasticsearch + kibana all in the same version.
Elasticsearch is not so important, but there is a stronger dependency between kibana and apm server.

Reply:
Typically I would say we need to mention minimum versions, but because 6.1 is beta, I think if we emphasize the "using the same version" of the stack, you're right, it probably doesn't need to be mentioned. I'll change the TIP to IMPORTANT.